cpp/include/cudf/lists/contains.hpp

```diff
@@ -74,6 +74,104 @@ std::unique_ptr<column> contains(
   cudf::column_view const& search_keys,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Create a column of bool values indicating whether each row in the `lists` column
```

Comment:
```suggestion
 * @brief Create a column of `bool` values indicating whether each row in the `lists` column
```

Reply:
Thanks for pointing this out. I've fixed it for all applicable overloads.